include/kyosu/details/cayleyify.hpp

```diff
@@ -0,0 +1,34 @@
+//======================================================================================================================
+/*
+  Kyosu - Complex Without Complexes
+  Copyright : KYOSU Contributors & Maintainers
+  SPDX-License-Identifier: BSL-1.0
+*/
+//======================================================================================================================
+#pragma once
+
+auto cayley_extend(auto f, auto z, auto... rs)
```

Comment:
care to explain what if of them do ?

Reply:
pb with sincos etc.-> cayley_extend2
pb with deta(k,z) etc. -> cayley_extend_rev
cayley_extend works for all other